sdk/include/crt/crtdefs.h

```diff
@@ -182,6 +182,15 @@
 
 #define __crt_typefix(ctype)
 
+#ifndef _STATIC_ASSERT
+  #ifdef __cplusplus
+    #define _STATIC_ASSERT(expr) static_assert((expr), #expr)
+  #elif defined(__clang__)
```

Comment:
GCC also has it, and the latest MSVC too (but there I'm not sure if it's enabled for C99)

Reply:
Added it for GCC. Doesn't seem to work for MSVC.